src/vm/ClrEtwAll.man

```diff
@@ -394,6 +394,8 @@
                           value="32" eventGUID="{BCF2339E-B0A6-452D-966C-33AC9DD82573}"
                           message="$(string.RuntimePublisher.AssemblyLoaderTaskMessage)">
                         <opcodes>
+                            <opcode name="ALCResolvingHandlerInvoked" message="$(string.RuntimePublisher.ALCResolvingHandlerInvokedOpcodeMessage)" symbol="CLR_ALC_RESOLVING_HANDLER_INVOKED_OPCODE" value="12"/>
```

Comment:
This does not follow the usual .NET naming conventions: 
- Use abbreviations for industry wide standards only (ALC is not one of those...)
- Use camel casing for abbreviations with more than 3 or more characters (ie it would be `Alc`)